### PR TITLE
#169258605 User can like and dislike

### PR DESCRIPTION
**Pull request by william000000**
#### What does this PR do?
User should be able to like and dislike accommodation
#### Description of Task to be completed?
- Like accommodation
- Dislike accommodation
- Test the functionality
#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-frontend.git`
- `cd team-odd-bn-frontend`
- `git checkout ft-like-Dislike-169258605`
- `npm install` packages
- `npm run dev`
-  Log in to the system then navigate to accommodations route
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#169258605](https://www.pivotaltracker.com/story/show/169258605)
#### Screenshots (if appropriate)
image

image

#### Questions:
- N/A

### CODE REVIEW

**Review by nziokaivy — APPROVED** [review]
Great work @william000000!

File: src/views/accommodation/ViewAccomodation.js [hunk 1]
```diff
@@ -9,21 +10,47 @@ import disLikeIcon from '../../assets/images/dislike_icon/thumbs-down_32.png';
...
+ componentDidMount(){
+   const {allAccomodation} = this.props.accommodation
+   this.setState({allAccomodation})
+ }
+ handleClick = async(like, items) =>{ 
+
+  if(like === "true"){
+   const passData =  await this.props.likeAction({like: true, id: items.id}) 
+    const updateView = await this.props.viewActionAccommodation()
+    return  { passData, updateView };
```
Comment (NiyongaboEric) [inline src/views/accommodation/ViewAccomodation.js]: You forgot to destructure the props.

File: src/__tests__/actions/accommodation/likeAndDislikeTest.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+              payload: {
+                message: 'dislike successfully',
+                payload: [{
+                  LikesDislikes: {
+                    dislikeCounter: 1,
+                    updatedAt: '2020-01-24T14:29:50.618Z',
+                    userId: 15,
+                  },
+                }],
+              },
```
Comment (NiyongaboEric) [inline src/__tests__/actions/accommodation/likeAndDislikeTest.js]: Move all mock data into `__mocks__` directory.

**Review by NiyongaboEric — CHANGES_REQUESTED** [review]
You did a good job @william000000. However, we can continue to improve the work. 
- checkout my feedback

File: src/views/accommodation/ViewAccomodation.js (hunk 1 above)
Comment (william000000) [reply src/views/accommodation/ViewAccomodation.js]: Thanks for the feedback @NiyongaboEric

**Review by NiyongaboEric — APPROVED** [review]

**Review by hezronkimutai — APPROVED** [review]
Really nice work @ willy.

**Review by shemaeric — APPROVED** [review]

**Pull request merged**